dts/bindings/gpio/arm,mps2-fpgaio-gpio.yaml

```diff
@@ -0,0 +1,21 @@
+title: ARM MPS2 FPGA GPIO
+
+description: >
+    This binding gives a base representation of the ARM MPS2 FPGA GPIO
```

Comment:
Could be something like

```yaml
description: >
    GPIO controller on ARM MPS2 FPGA
```

maybe. Says more with fewer words.

Thinking it would be best if the `description` text in bindings works as an answer to "what device is this?"